### PR TITLE
 Disable list of commands from playground repositories #897 - Commands Blacklisted

### PR DESCRIPTION
**Pull request by yashbudhia**
Based on the issue  Disable list of commands from playground repositories #897 in the main [dice](https://github.com/DiceDB/dice) repository , this PR adds a 
- 1. Blacklist.go file which contains a list of blacklist commands and logic for blacklisting.
- 2. Updated httpServer.go:  Implement the service layer and reject with a forbidden status
- 3. Also adds proper comments in the httpServer.go file for easier code readability

### Tests - 
Manually tested via postman for get as well as post routes and all blacklisted commands are throwing
errors in this fashion{"error": "ERR unknown command '{Command_name}"}

eg-
image

### Scope for improvement
  You can also test scenarios like:
- Sending a blacklisted command to see if it responds with the correct error.
-  Trying different HTTP methods (GET, POST) on various endpoints.

### CODE REVIEW

File: pkg/util/blacklist.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package helpers
+
+import (
+	"errors"
+	"strings"
+)
+
+var blacklistedCommands = []string{
```
Comment (lucifercr07) [inline pkg/util/blacklist.go:8]: Shall we keep it as map bool, so that checks would be faster?

File: pkg/util/blacklist.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	"FLUSHALL", "FLUSHDB", "DUMP", "ABORT", "AUTH", "CONFIG", "SAVE", "BGSAVE",
+	"BGREWRITEAOF", "RESTORE", "MULTI", "EXEC", "DISCARD", "QWATCH", "QUNWATCH",
+	"LATENCY", "CLIENT", "SLEEP", "PERSIST",
+}
+
+// IsBlacklistedCommand checks if a command is blacklisted
+func IsBlacklistedCommand(cmd string) error {
```
Comment (lucifercr07) [inline pkg/util/blacklist.go:15]: Please rename this to `BlockListedCommand`

**Review by lucifercr07 — CHANGES_REQUESTED** [review]
@yashbudhia thanks for contributing, please add tests for this.

**Comment by yashbudhia** [comment]
ok

**Comment by lucifercr07** [comment]
@yashbudhia please resolve conflicts once.

File: pkg/util/blacklist.go (hunk 2 above)
Comment (yashbudhia) [reply pkg/util/blacklist.go:15]: ok

**Comment by yashbudhia** [comment]
> @yashbudhia please resolve conflicts once.

I have resolved the conflicts

File: pkg/util/blacklist.go (hunk 1 above)
Comment (yashbudhia) [reply pkg/util/blacklist.go:8]: Yes, that would be faster, let me apply the changes

File: internal/server/http.go [hunk 3]
```diff
@@ -4,13 +4,14 @@ import (
 	"context"
 	"encoding/json"
 	"errors"
+	"fmt"  // Added to format error messages
```
Comment (lucifercr07) [inline internal/server/http.go:7]: Comment not required

**Review by lucifercr07 — CHANGES_REQUESTED** [review]
Please fix the build

**Comment by yashbudhia** [comment]
> Please fix the build

i have fixed the build and also removed the comments

**Review by yashbudhia — COMMENTED** [review]
desired changes applied

**Pull request merged**